setup.cfg

```diff
@@ -7,3 +7,7 @@ python-tag = py3
 [flake8]
 max-line-length=100
 exclude = configobj,passlib,cite.py,conf.py
+
+[tool:pytest]
+filterwarnings =
+  ignore:. is deprecated.*:DeprecationWarning
```

Comment:
That pattern is way too broad. It is going to suppress many warnings (even future ones) that could actually be relevant for us. What specific warnings are intended to be suppressed here?

Reply:
I realised that this practice is not good. I was trying to reduce the deprecation warning that were still appearing(snap short below to give you rough idea, you can reproduce the warning by running pytest in the forked branch, around 131 out of 1508 warning are still there) but the above approach is too broad as you said. Can you suggest what should I do for them?
![Screenshot from 2020-02-19 19-12-27](https://user-images.githubusercontent.com/26172160/74839755-cd493100-534b-11ea-8416-d722c6302cc1.png)

Reply:
I think it's fine to install a general warnings filter with a pattern that specifically matches " The indexing module is deprecated.".

Reply:
I am confused! Is there anything that I should be adding to deal with remaining warnings?

Reply:
It appears to me like every remaining warning would be covered by that?

Reply:
Okay, will do that and update you soon.